frontend/src/features/admin-form/template/TemplateFormProvider.tsx

```diff
@@ -101,63 +101,14 @@ export const TemplateFormProvider = ({
     [data?.form, data?.spcpSession],
   )
 
-  const { submitEmailModeFormMutation, submitStorageModeFormMutation } =
-    usePreviewFormMutations(formId)
-
+  // Forms are not submitted in template mode,
+  // hence this SubmitHandler does not contain submitEmailModeFormMutation and submitStorageModeFormMutation
   const handleSubmitForm: SubmitHandler<FormFieldValues> = useCallback(
```

Comment:
We may not need this function at all right? 

Should we update the context return type to be either a function or explicitly undefined (so devs will not accidentally forget the function), and use the presence of the function as a source of truth of whether the submit button should be disabled or not? 

This way we do not need a new variable

Reply:
[656ff12](https://github.com/opengovsg/FormSG/pull/5443/commits/656ff12b7b65076e95c6cc1199bf856d50e05551)